test/gtest/ucp/test_ucp_worker.cc

```diff
@@ -220,34 +226,31 @@ class test_ucp_worker_discard : public ucp_test {
         m_destroyed_ep_count++;
     }
 
-    static ep_test_info_t* ep_test_info_get(uct_ep_h ep) {
-        ep_test_info_t *test_info_p;
+    static ep_test_info_t& ep_test_info_get(uct_ep_h ep) {
         ep_test_info_map_t::iterator it = m_ep_test_info_map.find(ep);
 
         if (it == m_ep_test_info_map.end()) {
-            ep_test_info_t test_info = {};
+            ep_test_info_t test_info;
 
             m_ep_test_info_map.insert(std::make_pair(ep, test_info));
-            test_info_p = &m_ep_test_info_map.find(ep)->second;
-        } else {
-            test_info_p = &it->second;
+            it = m_ep_test_info_map.find(ep);
         }
 
-        return test_info_p;
+        return it->second;
     }
 
     static unsigned
     ep_test_info_flush_inc(uct_ep_h ep) {
-        ep_test_info_t *test_info = ep_test_info_get(ep);
-        test_info->flush_count++;
-        return test_info->flush_count;
+        ep_test_info_t &test_info = ep_test_info_get(ep);
+        test_info.flush_count++;
+        return test_info.flush_count;
```

Comment:
minor: can be shorter
return ++ep_test_info_get(ep).flush_count;

Reply:
done